jenkins-integration/beaker/install/shared/81_configure_jruby_jar.rb

```diff
@@ -0,0 +1,29 @@
+require 'puppet/gatling/config'
+
+test_name 'Configure jruby jar path'
+
+def service_config_name(service_name)
```

Comment:
Seems like it would be nice to put the following methods into the ./lib/puppet/gatling/config.rb helper file so that the configure_java_args and configure_jruby_jar code could reuse most of the implementation:

* service_config_name
* set_bash_variable
* call to get_puppet_server_service_name_from_env
* call to service_config_name

Maybe that way, this file could just be shortened to:

```ruby
require 'puppet/gatling/config'

test_name 'Configure jruby jar path'

set_bash_variable(master, service_config, 'JRUBY_JAR', ENV['PUPPET_GATLING_JRUBY_JAR'])
```

And the java_args one could be shortened to something similar.  WDYT?